src/pages/ipPools/components/PoolForm.js

```diff
@@ -41,6 +63,19 @@ export class PoolForm extends Component {
                 />
               </AccessControl>
             }
+
+            {!editingDefault && Boolean(overflowPools.length) &&
+              <AccessControl condition={isAccountUiOptionSet('ip_auto_warmup', false)}>
+                <Field
+                  name='auto_warmup_overflow_pool'
+                  label='Overflow Pool'
+                  component={SelectWrapper}
+                  options={overflowPools}
+                  helpText='If you are using the automated IP warmup feature, the overflow pool will be used when volume threshold for this pool has been reached.'
```

Comment:
Does this make sense?

```suggestion
                  helpText='With automatic IP warmup enabled, this pool will be used for excess sending volume to protect your IPs.'
```

Reply:
either way sounds good to me.